doc/releases/changelog-dev.md

```diff
@@ -179,6 +179,12 @@
 
 <h3>Deprecations 👋</h3>
 
+* The boolean functions provided by ``pennylane.operation`` are deprecated. See each function's docstring
+  for equivalent code. These include :func:`not_tape`, :func:`has_gen`, :func:`has_grad_method`,
```

Comment:
```suggestion
  for equivalent code to use instead. These include :func:`not_tape`, :func:`has_gen`, :func:`has_grad_method`,
```

Reply:
Merged the suggestion into the other suggestion.